### PR TITLE
[v2][www] Fix missing search sidebar for plugins/packages pages

### PR DESCRIPTION
**Pull request by ryanditjia**
Ref: https://github.com/gatsbyjs/gatsby/issues/5348#issuecomment-389814208

`PageWithSearchBar` component styles are taken from [master’s layout](https://github.com/gatsbyjs/gatsby/blob/master/www/src/layouts/index.js#L47-L121) and streamlined.

### CODE REVIEW

**Comment by m-allanson** [comment]
Refs: #5348

File: www/src/components/page-with-searchbar.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React, { Component, Fragment } from "react"
```
Comment (KyleAMathews) [inline www/src/components/page-with-searchbar.js]: Probably we should rename this component to something like "plugin-sidebar"

Comment (ryanditjia) [reply www/src/components/page-with-searchbar.js]: I agree that "searchbar" by itself is confusing. However, considering mobile view of plugins index (gatsbyjs.org/plugins) is just this "plugin-sidebar", would you be okay with "plugin-searchbar" instead?

Comment (KyleAMathews) [reply www/src/components/page-with-searchbar.js]: That sounds perfect.

**Comment by KyleAMathews** [comment]
Thanks!

**Pull request merged**